integration_tests/ctesque/src/sharedTest/java/android/app/UiAutomationTest.java

```diff
@@ -55,4 +66,23 @@ public void setRotation_freeze180_isPortrait() {
           });
     }
   }
+
+  private static void waitDisplayRotation(
+      Activity activity, int expectedRotation, long timeOutMilliSeconds) {
+    long startMs = SystemClock.uptimeMillis();
+    long currentMs = SystemClock.uptimeMillis();
+    while (currentMs - startMs <= timeOutMilliSeconds) {
+      Display display = activity.getWindowManager().getDefaultDisplay();
```

Comment:
Extract it outside the `while`? Or is there a change that the instance changes at each iteration?